DFC.ServiceTaxonomy.ApiFunction/Function/Execute.cs

```diff
@@ -62,6 +69,9 @@ public Execute(IOptionsMonitor<ServiceTaxonomyApiSettings> serviceTaxonomyApiSet
                 var cypherPathParameters = GetCypherPathParameters(cypherModel, req.Path, log);
                 var cypherQueryParameters = GetCypherQueryParameters(cypherModel, req.Query, await requestBodyTask, log, cypherPathParameters);
 
+                //Add the host in to all cypher queries
+                cypherQueryParameters.Add("host", host);
```

Comment:
I think `X-Forwarded-Host` will just contain the domain.
I think we'll have to construct the whole initial part of the url, including the scheme and the function name.

Reply:
Should give "test.com" so we only need to apply the scheme? As Job Profile URLs will then have http://test.com/job-profile/bottler for example